### ISSUE TITLE
Folder 'dcdata'

### ISSUE THREAD

**Issue by pramodhrachuri** [zhr1201/deep-clustering#13]
Hi there,
In the file **train_net.py**, you are trying to access **1.pkl, 2.pkl,...12.pkl** from a folder **dcdata**
But we haven't generated any such files. Ofc we have generated _val.pkl_ using _datagenerator.py_

Please help me understand what these are for and how to generate them.

**Comment by zhr1201** [comment]
Sorry for the misunderstanding, I meant train.py should be used to generate 1.pkl, 2.pkl...

**Comment by clisac** [comment]
Hello, I have the same question. 
There is no train.py provided in you code. What do you mean by that?

**Comment by pramodhrachuri** [comment]
@zhr1201 
Thank you for replying. Please guide me through this issue.

Did you mean _datagenerator.py_ ?
And what exactly are **1.pkl, 2.pkl, ....**, how to generate them?

**Comment by vzxxbacq** [comment]
I guess this folder contains features of owner's train audio. Maybe you should extract your train audio's features and write down these features as *.pkl file, then put their path into datagenerator class.

**Comment by janhavi028** [comment]
@vzxxbacq there will be one pkl file for one audio or there will be a single pkl file containing the features of the whole data set?

**Comment by vzxxbacq** [comment]
Hello @janhavi028 ,  I think we can write them in any number of files with any number of features. In `train_net.py` author pass 12 files to the model and check function `re_init()` in `datagenerator.py` we know author extract features and put them into one pkl file.

**Comment by janhavi028** [comment]
@vzxxbacq Thank you!

**Comment by tangchenjun** [comment]
@vzxxbacq Hi! Do you mean I should  use datagenerator.py  to get 12 pkl files for 12 single persons and  one  pkl file for the whole 12 persons?

**Comment by tangchenjun** [comment]
@pramodhrachuri  Hey! How did you generate the 12 pkl files? Do they belong to 12 different speakers? I am confused with that,thank you!

**Comment by zhr1201** [comment]
@tangchenjun Not necessarily, I generated each file using the same scheme and raw data with random sampling for 12 times.

**Comment by bailiangze** [comment]
请问意思是，用同样的数据，利用datagenerator.py产生12个pkl文件？我想问如果采用timit语音库，每个说话人大概只有5语音？怎么使用啊
